web/views.py

```
@@ -4443,21 +4461,17 @@ def team_goal_detail(request, goal_id):
 
-    # Get existing team members to exclude from invitation
     existing_members = goal.members.values_list("user_id", flat=True)
 
-    # Handle inviting new members
     if request.method == "POST":
         form = TeamInviteForm(request.POST)
         if form.is_valid():
-            # Check for existing invites using the validated User object
             if TeamInvite.objects.filter(
-                goal__id=goal.id, recipient=form.cleaned_data["recipient"]  # Changed to use User object
+                goal__id=goal.id, recipient=form.cleaned_data["recipient"] 
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)